doc/docs/technical-documentation/systemd.md

```diff
@@ -20,4 +20,4 @@ When systemd is enabled, WSL tries synchronizes launching processes with systemd
 To improve compatibility with systemd, WSL creates various systemd configuration files during boot (under `/run`). These configurations files are used to:
 
 - Protect the WSL [binfmt interpret](interop.md) from being deleted by `systemd-binfmt.service`
-- Protect the X11 socket from being deleted by `systemd-tmpfiles.serviceè 
```

Comment:
This was identified by a rule complaining about non-ascii characters...

Reply:
I think this should be 'systemd-tmpfiles-setup.service'

@oneblue to confirm.

Reply:
I think you're right, so I've added a change for that: 85a686eadd424cf65318b97a1a80a9e30b6818d1

(It's trivial for me to drop things if we're wrong...)